spec/campaigns_spec.rb

```diff
@@ -3,17 +3,16 @@
 require 'spec_helper'
 
 describe UplandMobileCommonsRest::Campaigns do
-  let(:client) { UplandMobileCommonsRest::Client.new(username: 'username', password: 'password') }
+  let(:client) { UplandMobileCommonsRest::Client.new(api_key: 'abcde-123456') }
 
-  subject { UplandMobileCommonsRest::Campaigns.new(client: client) }
+  subject { UplandMobileCommonsRest::Campaigns.new(client:) }
 
   it 'should initialize' do
     expect(subject.client).to eq(client)
   end
 
   context 'with credentials' do
-    let(:username) { 'username' }
-    let(:password) { 'password' }
+    let(:api_key) { 'abcde-123456' }
```

Comment:
What is this variable used for?

Reply:
It's used for stubbing the request's header with webmock here: https://github.com/controlshift/upland_mobile_commons_rest/blob/api_token_auth/spec/support/stub_requests.rb#L5